### ISSUE TITLE
[PVC] Segfault in to_sparse for torch.complex128

### ISSUE THREAD

**Issue by Silv3S** [intel/torch-xpu-ops#3215]
### 🐛 Describe the bug

Cases:
~~op_ut,third_party.torch-xpu-ops.test.xpu.test_meta_xpu.TestMetaXPU,test_meta_outplace_to_sparse_xpu_complex128~~

This issue is minimal reproduction of original test_meta_outpalce_to_sparse_xpu_complex128. Test works for all dtypes except complex128 for which segmentation fault is observed. Segfault is specific to PVC, as BMG works without issues.

Small modifications to repro code - changing seed, using different shape of input tensor makes test pass. 

```Python
import torch
# from torch.testing import make_tensor
# from torch.testing._internal.common_utils import set_rng_seed

def example_input():
    # set_rng_seed(2877894469)    # Need this seed for segfault reproduction
    # print(make_tensor((5, 5), device="xpu", dtype=torch.complex128, requires_grad=False))

    return torch.tensor([
        [ 0.5684-4.9757j, -2.7630+3.7411j, -1.0964-2.4930j,  1.9511+6.9829j, 8.6619+6.0138j],
        [-5.1101+6.7623j,  7.8563+6.5123j,  5.7766+4.4107j, -0.4579-6.7331j, -4.6525-5.8946j],
        [ 7.6683-8.4361j,  5.2232-5.1585j, -2.3929-8.2543j,  6.1614-6.6898j, 1.6617+0.3332j],
        [ 5.8428+7.9773j,  8.7044+4.4861j, -7.7409+3.8714j,  1.6632-2.2653j, 3.1204-1.8052j],
        [-8.7791-1.5369j, -6.6341-2.3336j, -7.6261-5.0738j,  3.9731-5.9966j, 6.3050+5.7251j]],
        device='xpu:0', dtype=torch.complex128)

# PASS | Add del s1 before calling to_sparse on t2, which seems to prevent the segfault.
def test_del_workaround():
    t1 = example_input()
    s1 = t1.to_sparse()
    t2 = example_input()
    del s1
    s2 = t2.to_sparse(1)

# PASS | Call to_sparse on t2 before calling to_sparse on t1, which seems to prevent the segfault.
def test_order_workaround():
    t1 = example_input()
    t2 = example_input()
    s1 = t1.to_sparse()
    s2 = t2.to_sparse(1)

# PASS | Remove assignment to s1 variable, which seems to prevent the segfault.
def test_assignment_workaround():
    t1 = example_input()
    t1.to_sparse()
    t2 = example_input()
    s2 = t2.to_sparse(1)

# FAIL | Segfault observed only in this case
def test_minimal_segfault():
    t1 = example_input()
    s1 = t1.to_sparse()
    t2 = example_input()
    s2 = t2.to_sparse(1)
```

Error message
```
Segmentation fault from GPU at 0x5119ce0360000, ctx_id: 1 (CCS) type: 0 (NotPresent), level: 4 (PML5), access: 0 (Read), banned: 1, aborting.
Segmentation fault from GPU at 0x5119ce0360000, ctx_id: 1 (CCS) type: 0 (NotPresent), level: 4 (PML5), access: 0 (Read), banned: 1, aborting.
Abort was called at 288 line in file:
./shared/source/os_interface/linux/drm_neo.cpp
Fatal Python error: Aborted
```

### Versions

[level_zero:gpu][level_zero:0] Intel(R) oneAPI Unified Runtime over Level-Zero, Intel(R) Data Center GPU Max 1550 12.60.7 [1.6.33578+42]
[opencl:gpu][opencl:1] Intel(R) OpenCL Graphics, Intel(R) Data Center GPU Max 1550 OpenCL 3.0 NEO  [25.18.33578]
